ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_service.proto

```diff
@@ -46,11 +46,18 @@ message SubmitAndWaitRequest {
 message SubmitAndWaitForTransactionIdResponse {
   // The id of the transaction that resulted from the submitted command.
   // Must be a valid LedgerString (as described in ``value.proto``).
-  // Required
+  // In the Ledger API 3.0 this field will be removed in favor of the update_id field.
+  // Required for single-domain API, ignored for multi-domain API
```

Comment:
Can we say empty instead of ignored ?

Reply:
sure

Reply:
oh wait a second: do you want to enforce validation that the transaction_id field is NOT filled, in case of the update_id field is expected?
I have no strong opinion, but this is more strict than what I had in mind for the transitioning period (until 3.0)

Reply:
deprecated